pallets/subtensor/src/utils.rs

```diff
@@ -140,6 +140,16 @@ impl<T: Config> Pallet<T> {
         WeightsMinStake::<T>::put(min_stake);
         Self::deposit_event(Event::WeightsMinStake(min_stake));
     }
+    pub fn set_target_stakes_per_interval(target_stakes_per_interval: u64) {
+        TargetStakesPerInterval::<T>::set(target_stakes_per_interval)
+    }
+    pub fn set_stakes_this_interval_for_hotkey(hotkey: &T::AccountId, stakes_this_interval: u64) {
+        TotalHotkeyStakesThisInterval::<T>::insert(hotkey, stakes_this_interval);
+    }
+    pub fn reset_stakes_and_unstakes_this_interval() {
+        // This removes all key-value pairs from the storage map
+        let _ = TotalHotkeyStakesThisInterval::<T>::clear(u32::MAX, None);
```

Comment:
Need to be careful about this one, how many hotkey stakes do we expect to have in each interval? If it's a lot, it may consume a lot of compute from the chain.

Reply:
Each hotkey can only stake or unstake once per interval, defined as a duration of 360 blocks or equivalently, a 72-minute period, given a block time of 12 seconds.

My assumption writing this is that there won't be "too many" stakes in any given 72 minuet window especially considering the financial cost to the staker per stake.